internal/naming/port.go

```diff
@@ -9,6 +9,11 @@ import (
 	"strings"
 )
 
+const (
+	TargetAllocatorPort       = 80
```

Comment:
nit: can we change the name to: 
```suggestion
	TargetAllocatorServicePort       = 80
```